sandpack-client/src/utils.ts

```diff
@@ -17,6 +17,15 @@ export function nullthrows<T>(value?: T | null, err = "Value is nullish"): T {
   return value;
 }
 
+export function codeToString(code: string | Uint8Array): string {
```

Comment:
There is also `readBuffer` function in node runtime utils that do the same thing but using `Buffer.from(Uint8Array).toString('utf-8')` instead of `TextDecoder`. Should I remove this and update `readBuffer` implementation to use `TextDecoder` so it can be used outside nodebox?